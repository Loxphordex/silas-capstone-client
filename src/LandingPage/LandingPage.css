.landing-button {
  position: relative;
  top: 20px;
  margin: 9px auto;
}

.landing-page-info {
  background-color: rgb(32, 43, 92);
  color: rgb(218, 218, 218);
  border-radius: 7%;
  text-align: center;
  height: 190px;
  width: 92%;
  max-width: 300px;
  font-size: 15px;
  padding-top: 4em;
  margin: auto;
  margin-top: 30px;
}

.landing-page-info p {
  margin: 0 17%;
}

#landing-nav {
  text-align: right;
  background-color: white;
  margin-top: 25px;
  width: 90%;
}

#landing-header {
  font-size: 30px;
  color: white;
}

#start-writing {
  font-family: 'Satisfy', cursive;
  font-size: 28px;
  background-color: rgb(155, 212, 48);
  width: 92%;
  border-radius: 21px;
}

#start-writing a {
  color: rgb(76, 46, 160);
}

#register {
  font-size: 28px;
  background-color: rgb(209, 84, 53);
  width: 92%;
  border-radius: 21px;
}

#register a {
  color: white;
}

.sub-info {
  text-align: left;
  width: 90%;
  margin: 22px auto 10px auto;
}

.sub-info h3 {
  margin: 0;
  margin-top: 10px;
  font-size: 14px;
}

.sub-info p {
  margin: 0;
  font-size: 12px;
  width: 70%;
}