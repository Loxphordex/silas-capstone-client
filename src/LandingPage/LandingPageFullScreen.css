@media only screen and (min-width: 600px) {
  #landing-login {
    font-size: 17px;
  } 
  .landing-page {
    min-width: 700px;
    margin: 0 20%;
  }
  .landing-page h1 {
    position: relative;
    left: 6px;
    top: 48px;
  }
  .landing-page-info {
    display: none;
  }
  .landing-full-info {
    display: inline-block;
    min-width: 700px;
  }
  .landing-full-info h1 {
    color: white;
  }
  #landing-full-head-block {
    display: inline-block;
    background-color: rgb(1, 97, 87, 38);
    color: rgb(218, 218, 218);
    border-radius: 7%;
    text-align: center;
    height: 190px;
    width: 42%;
    max-width: 300px;
    font-size: 18px;
    padding-top: 4em;
    margin: auto 5px;
    margin-top: 30px;
  }
  #landing-full-info-block {
    position: relative;
    top: -47px;
    display: inline-block;
    background-color: rgb(231, 215, 187, 98);
    color: rgb(46, 46, 46);
    border-radius: 7%;
    text-align: left;
    height: 161px;
    width: 36%;
    max-width: 300px;
    font-size: 18px;
    padding-top: 4em;
    margin: auto 5px;
    margin-top: 30px;    
  }
  #landing-full-info-block p {
    max-width: 80%;
    margin: auto;
  }
  .sub-info {
    margin: 40px auto 10px auto
  }
  .sub-info h3 {
    font-size: 15px;
  }
  .sub-info p {
    font-size: 14px;
    margin-bottom: 13px;
  }
  #start-writing {
    width: 48%;
  }
  #register {
    width: 40%;
  }
}