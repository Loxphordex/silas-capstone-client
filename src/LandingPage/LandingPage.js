import React from 'react'
import { Link } from 'react-router-dom'
import Nav from '../Nav/Nav'
import './LandingPage.css'

export default class LandingPage extends React.Component {
  render() {
    return (
      <div className='landing-page'>

        <nav role='navigation' id='landing-nav'>
          <div id='landing-login'><Link to='/login'>Login</Link></div>  
        </nav>

        <div className='landing-page-info'>
          <h1 id='landing-header'>Quoter</h1>
          <p>Write about anything. See what the greatest minds have to say about it.</p>
        </div>

        <div id='start-writing'><Link to='/entry'>Start Writing</Link></div>
        <div id='register'><Link to='/register'>Sign Up</Link></div>

      </div>
    )
  }
}