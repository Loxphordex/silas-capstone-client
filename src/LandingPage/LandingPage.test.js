import React from 'react';
import LandingPage from './LandingPage'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import Enzyme from 'enzyme'
import Adapter from 'enzyme-adapter-react-16'

Enzyme.configure({ adapter: new Adapter() })

describe('Landing Page', () => {
    it('renders the complete page', () => {
    const wrapper = shallow(<LandingPage />)
    expect(toJson(wrapper)).toMatchSnapshot()
  })
})