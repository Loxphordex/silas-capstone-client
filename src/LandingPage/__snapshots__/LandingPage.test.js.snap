// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Landing Page renders the complete page 1`] = `
<div
  className="landing-page"
>
  <nav
    id="landing-nav"
    role="navigation"
  >
    <div
      id="landing-login"
    >
      <Link
        to="/login"
      >
        Login
      </Link>
    </div>
  </nav>
  <div
    className="landing-page-info"
  >
    <h1
      id="landing-header"
    >
      Quoter
    </h1>
    <p>
      Write about anything. See what the greatest minds have to say about it.
    </p>
  </div>
  <div
    className="landing-full-info"
  >
    <div
      className="full-block"
      id="landing-full-head-block"
    >
      <h1
        id="full-header"
      >
        Quoter
      </h1>
    </div>
    <div
      className="full-block"
      id="landing-full-info-block"
    >
      <p>
        Write about anything. See what the greatest minds have to say about it.
      </p>
    </div>
  </div>
  <div
    className="sub-info"
  >
    <h3>
      Inspiration for writing, from writing
    </h3>
    <p>
      Quoter scans what you write and finds quotes to match.
    </p>
    <h3>
      A treatment for writer's block
    </h3>
    <p>
      Discover new authors as well as new perspecives.
    </p>
  </div>
  <div
    className="landing-button"
    id="start-writing"
  >
    <Link
      to="/entry"
    >
      Start Writing
    </Link>
  </div>
  <div
    className="landing-button"
    id="register"
  >
    <Link
      to="/register"
    >
      Sign Up
    </Link>
  </div>
</div>
`;
